package.json

```diff
@@ -7,13 +7,35 @@
     "node": "18 || 20"
   },
   "scripts": {
-    "create-workspace": "community-cli workspace create"
+    "create-workspace": "community-cli workspace create",
+    "prepare": "husky"
```

Comment:
What's the best way to enable husky automatically? I know we set this up in the main repo but can't rememeber how we did it.

Reply:
This is it, isn't it!

Reply:
But when does `prepare` get called? Cos if I run `yarn install` at the root it doesn't automatically run this?

Reply:
https://classic.yarnpkg.com/lang/en/docs/package-json/#toc-scripts

"For compatibility reasons, scripts called install, postinstall, prepublish, and prepare will all be called after your package has finished installing"

Not sure what "install" means for a root package json, but afaik it works. 😅

Reply:
https://docs.npmjs.com/cli/v10/using-npm/scripts#:~:text=Runs%20BEFORE%20the%20package%20is%20packed%2C%20i.e.%20during%20npm%20publish%20and%20npm%20pack

Reply:
but we're not on `yarn classic`? I guess that's the reason?

Reply:
@benjdlambert good catch - this wasn't working since we're not using yarn 1. According to to the [husky docs](https://typicode.github.io/husky/how-to.html#manual-setup) this should be hooked up to the `postinstall` script instead when using yarn - updated this accordingly

Reply:
hmm my understanding is that `postinstall` should be compatible with all the above scenarios?